packages/client/src/StreamrClient.ts

```diff
@@ -124,7 +158,14 @@ export class StreamrClient {
     // --------------------------------------------------------------------------------------------
 
     /**
+     * Subscribes to a stream partition in the network.
+     *
      * @category Important
+     *
+     * @param options - the stream or stream partition to subscribe to,
```

Comment:
Always a partition, not the whole stream. (Also line `193`)

Maybe we should just say that by default we use the first partion. That fact is now somehow hidden, maybe because we planned (or are still planning?) to change the semantics of how subscribe works if no partition is given.